src/sssom/context.py

```diff
@@ -97,7 +97,7 @@ def get_default_metadata() -> Metadata:
     :return: Metadata
     """
     contxt = get_jsonld_context()
```

Comment:
this is the sssom schema prefix map i guesss

Reply:
Yes ... And this will be gone in the next PR when we entirely rely on bioregistry.

Reply:
Ok, but if we do this, I want a hard coded test in the testing framework that ensures that the whenever the epm is updated, the built-in prefixes are _exactly_ what we expect them to be (which is what is in sssom context).